safety/cli.py

```diff
@@ -92,6 +92,18 @@ def check(key, db, json, full_report, bare, stdin, files, cache, ignore, output,
     except DatabaseFileNotFoundError:
         click.secho("Unable to load vulnerability database from {db}".format(db=db), fg="red", file=sys.stderr)
         sys.exit(-1)
+    except NetworkConnectionError:
+        click.secho("Check your network connection, unable to reach the server", fg="red", file=sys.stderr)
+        sys.exit(-1)
+    except RequestTimeoutError:
+        click.secho("Check your network connection, the request timed out.", fg="red", file=sys.stderr)
+        sys.exit(-1)
```

Comment:
If we create a pattern for passing default messages to our error classes, you can just reuse the error message of the thrown exception here and maybe only catch the base class like `DatabaseFetchError`.